test/common/schemaform/FormApp.unit.spec.jsx

```diff
@@ -128,7 +127,6 @@ describe('Schemaform <FormApp>', () => {
           currentLocation={currentLocation}
           loadedStatus={LOAD_STATUSES.pending}
           prefillStatus={PREFILL_STATUSES.pending}
-          isLoggedIn
```

Comment:
These weren't used before. I actually added this prop (back) and it's only used in the auto-load bits.